Podfile.lock

```diff
@@ -20,6 +21,7 @@ PODS:
 DEPENDENCIES:
   - Expecta
   - HockeySDK (~> 3.6.2)
+  - Keys (from `/Users/ash/bin/Tropos/Pods/CocoaPodsKeys/`)
```

Comment:
Since this references your machine, does this work when other people clone down the app?